netaddr.go

```diff
@@ -1465,14 +1465,14 @@ func (r IPRange) contains(addr IP) bool {
 	return r.from.Compare(addr) <= 0 && r.to.Compare(addr) >= 0
 }
 
-// less returns whether r is "before" other. It is before if r.From() is
-// before other.From(), or if they're equal, the shorter range is
-// before.
+// less returns whether r is "before" other. It is before if r.From()
+// is before other.From(), or if they're equal, the larger range comes
```

Comment:
Let's split into two sentences. "...is before other.From(). If they're equal, ..."

Reply:
Done.